modules/core/src/main/java/org/apache/synapse/aspects/flow/statistics/collectors/CloseEventCollector.java

```diff
@@ -98,6 +103,12 @@ public static void closeFlowForcefully(MessageContext messageContext, boolean er
             } else {
                 addEventAndCloseFlow(messageContext, endFlowEvent);
             }
+
+            if (isOpenTracingEnabled()) {
+				OpenTracingManagerHolder.getOpenTracingManager().getHandler()
+						.handleCloseFlowForcefully(dataUnit, messageContext);
+			}
```

Comment:
Formatting issue

Reply:
Done in https://github.com/wso2/wso2-synapse/pull/1466/commits/f667809fdde325456df56a477b83e6ef731107ca